nectar/src/config/serde/bitcoin_amount.rs

```diff
@@ -31,7 +31,7 @@ where
                 Some(value) => {
                     let amount = Amount::from_btc(value).map_err(|error| {
                         serde::de::Error::custom(format!(
-                            "Could not deserialize bitcoin amount: {:?}",
+                            "Could not deserialize bitcoin amount: {:#}",
                             error
                         ))
                     })?;
```

Comment:
This could just be `Amount::from_btc(value).map_err(Error::custom)?`, not sure if we need the extra string in there.